CHANGELOG.md

```diff
@@ -79,7 +80,9 @@
   from Hex.
 
 ## v0.12.0-rc3 - 2020-10-24
+
 ## v0.12.0-rc2 - 2020-10-24
+
 ## v0.12.0-rc1 - 2020-10-24
```

Comment:
ah whoops, these extra additions are from prettier formatting the markdown file, want me to remove them?

Reply:
Nah looks good